### PR TITLE
Adding Arabic locale for the first time

### PR DESCRIPTION
**Pull request by vormia**

### CODE REVIEW

**Comment by NateWr** [comment]
Thanks! @mtub can you take a look?

File: locale/ar_IQ/locale.xml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * plugins/themes/bootstrap3/locale/ar_IQ/locale.xml
+  *
+  * Copyright (c) 2014-2017 Simon Fraser University Library
```
Comment (asmecher) [inline locale/ar_IQ/locale.xml]: @vormia, can you change 2017 to 2019 on this line and the next, and change "Simon Fraser University Library" to "Simon Fraser University"?

Comment (vormia) [reply locale/ar_IQ/locale.xml]: Done

**Comment by asmecher** [comment]
Thanks again, @vormia!

**Pull request merged**